src/main/resources/META-INF/pluginIcon_dark.svg

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
+ "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<svg xmlns="http://www.w3.org/2000/svg"
+     width="256.000000pt" height="256.000000pt" viewBox="0 0 256.000000 256.000000"
+     preserveAspectRatio="xMidYMid meet">
+<metadata>
+Created by potrace 1.15, written by Peter Selinger 2001-2017
+</metadata>
+<g transform="translate(0.000000,256.000000) scale(0.100000,-0.100000)"
+fill="#000000" stroke="none">
+<path d="M1205 2536 c-16 -8 -86 -71 -155 -140 l-125 -126 148 -148 148 -149
+53 5 c116 9 201 -77 194 -199 l-3 -59 134 -134 135 -135 60 0 c34 1 73 -3 87
+-9 147 -58 157 -273 17 -341 -141 -68 -303 55 -268 204 l9 40 -134 135 -135
+135 0 -353 0 -353 36 -31 c54 -48 67 -81 62 -159 l-5 -67 133 -133 134 -134
+396 395 c222 221 402 409 410 427 20 45 17 107 -5 151 -24 46 -1134 1154
+-1178 1176 -39 19 -107 20 -148 2z" fill="#F1502F"/>
+<path d="M423 1768 c-216 -216 -397 -406 -403 -421 -15 -41 -12 -103 6 -140
+22 -44 1130 -1154 1176 -1178 41 -21 101 -25 145 -9 15 6 85 68 156 138 l127
+127 -137 138 -138 137 -42 -6 c-53 -8 -105 9 -150 49 -86 75 -85 199 1 275 20
+17 41 32 46 32 6 0 10 134 10 355 0 195 -3 355 -8 355 -4 0 -69 -62 -145 -138
+l-138 -138 9 -40 c14 -64 -4 -120 -53 -169 -117 -117 -315 -37 -315 128 0 112
+85 192 196 185 l54 -3 141 141 140 141 3 74 2 74 -140 143 c-78 78 -143 142
+-146 142 -3 0 -181 -177 -397 -392z" fill="#c1d3ff"/>
```

Comment:
Edited manually? ;)

Reply:
@mkondratek consider making the color in the lower half the same as the `Git Machete` text (the current one is too bluish)

Reply:
![image](https://user-images.githubusercontent.com/19799111/80455202-a13dc380-892b-11ea-874d-51fce777f655.png)